__tests__/fixtures/tailwind-output.css

```diff
@@ -412,7 +414,7 @@ ul {
  */
 
 html {
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji"; /* 1 */
+  font-family: 'Inter', sans-serif; /* 1 */
```

Comment:
You shouldn’t kill the whole font stack here, but just add `Inter` in front of `-apple-system`. That way, if Inter fails to load, the user still gets to see the normal system fonts.

Reply:
I agree. I purposely simplified it for this proof of concept. After discussing how to load the font if it’s needed and the addition of the optional config, I planned on changing the stack to include more sensible fallbacks. 

It also makes me wonder, though it’s unrelated to Tailwind, if browsers should use the system font stack in place of Helvetica or Arial when sans-serif is used. That a conversation for another place entirely though. 

What are your thoughts on the implementation? Is loading Inter from a cdn or external source better than having the font in Tailwind core and copying font files to the users build directory? I’m still torn here.